integration-cli/fixtures.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
I could probably do some.. but these are relying on things set in this package (dockerBinary, daemonPlatform, and a few other things).
Not exactly out of the realm to change these, but not sure about the scope of it.